app/notify_client/service_api_client.py

```diff
@@ -1,19 +1,29 @@
 from __future__ import unicode_literals
 
-from notifications_python_client.notifications import NotificationsAPIClient
+from notifications_python_client import __version__
+from notifications_python_client.notifications import BaseAPIClient
 
 
-class ServiceApiClient:
+class ServiceApiClient(BaseAPIClient):
     def __init__(self):
-        self.api_client = None
+        super().__init__("a" * 73, "b")
 
     def init_app(self, application):
-        self.api_client = NotificationsAPIClient(base_url=application.config["API_HOST_NAME"], api_key="a" * 75)
-        self.api_client.service_id = application.config["ADMIN_CLIENT_USER_NAME"]
-        self.api_client.api_key = application.config["ADMIN_CLIENT_SECRET"]
+        self.base_url = application.config["API_HOST_NAME"]
+        self.service_id = application.config["ADMIN_CLIENT_USER_NAME"]
+        self.api_key = application.config["ADMIN_CLIENT_SECRET"]
+        self.route_secret = application.config["ROUTE_SECRET_KEY_1"]
```

Comment:
On the PaaS, we don't appear to be setting the `ROUTE_SECRET_KEY_1` in the manifest file. This means that `application.config["ROUTE_SECRET_KEY_1"]` is an empty string. 

Is that what you are after? It probably would be fine because the requests on the PaaS will still go out to cloudfront as they use the external address and then the header should be added there.